src/test/java/posters/cucumber/support/OrderSupport.java

```diff
@@ -33,24 +33,24 @@ public OrderSupport(GlobalStorage storage)
     @Given("^new user with \"([^\"]*)\", \"([^\"]*)\", \"([^\"]*)\", \"([^\"]*)\" is registered and logged in$")
     public void registerAndLogIn(String firstName, String lastName, String email, String password)
     {
-        RegisterPage registerPage = OpenPageFlows.registerPage();
+        var registerPage = OpenPageFlows.registerPage();
         registerPage.isExpectedPage();
         storage.user = new User(firstName, lastName, email, password);
         registerPage.sendRegisterForm(firstName, lastName, email, password, password);
-        LoginPage loginPage = registerPage.userMenu.openLogin();
+        var loginPage = registerPage.userMenu.openLogin();
         loginPage.isExpectedPage();
         loginPage.sendLoginform(email, password);
     }
 
     @Then("^all the products are to find in order history$")
     public void validateOrderInOrderHistory()
     {
-        AccountOverviewPage accountOverview = new HomePage().userMenu.openAccountOverview();
-        accountOverview.isExpectedPage();
-        OrderHistoryPage orderHistory = accountOverview.openOrderHistory();
+        var accountOverviewPage = new HomePage().userMenu.openAccountOverview();
+        accountOverviewPage.isExpectedPage();
+        var orderHistoryPage = accountOverviewPage.openOrderHistory();
         for (Product product : storage.products)
```

Comment:
Be consistent for structures like:
```
for (Product product : storage.products){...}
```
Decide if we use `Product product :` or `var product :` and adjust it at all occurrences.
BTW: I would use  `Product product :`  since this is a little bit more readable if you are working on class fields.